src/common/components/fix-instruction-panel.tsx

```diff
@@ -47,8 +48,8 @@ export const FixInstructionPanel = NamedFC<FixInstructionPanelProps>('FixInstruc
 
     return (
         <div>
-            {props.renderTitleElement(title, 'insights-fix-instruction-title')}
-            <ul className="insights-fix-instruction-list">{renderInstructions(props.checkType)}</ul>
+            {props.renderTitleElement(title, null)}
```

Comment:
It looks like this is the only spot where the function in question is invoked; should the second parameter be removed?

Reply:
The function definition doesn't have this as an optional param and everywhere this is used, its being passed a classname, so didn't touch the definition.

Reply:
I'm saying that I believe this callsite is the only non-test callsite where a classname was ever passed to any concrete implementation, so I think it would make sense to remove the classname param from the implementations now that it's obsolete

Reply:
Done.